### PR TITLE
Simplify DAG pagination, fix Block timestamps in DAG page, fix parsing of boolean query params

### PR DESCRIPTION
**Pull request by cblage**
This PR simplifies the pagination in the DAG blocks page, it fixes the timestamps shown for DAG blocks and fixes the parsing of boolean query params in a couple of places.

### CODE REVIEW

File: models/dag_block.js [hunk 1]
```diff
@@ -38,4 +38,7 @@ DAGBlock.statics.fromRPC = function fromRPC(json) {
   return new this(json);
 };
 
+DAGBlock.index({ level: 1 }, { background: true });
```
[the remark this answers is not in the document]
Comment (cblage) [reply models/dag_block.js:41]: good idea! I'll move it

File: pages/dag_blocks.js [hunk 2]
```diff
@@ -1,222 +1,81 @@
...
+          style={{
+            paddingLeft: 5,
+            paddingRight: 5,
+            paddingTop: 5,
+            paddingBottom: 0,
+            margin: 0,
+            backgroundColor: '#0f1517',
+          }}
+        >
+          <DAG dagBlocks={blocks} reverse={true} pbftBlocks={pbftBlocks} />
```
[the remark this answers is not in the document]
Comment (cblage) [reply pages/dag_blocks.js:67]: yep, good catch

File: pages/dag_blocks.js [hunk 3]
```diff
@@ -1,222 +1,81 @@
...
-  const pages = Math.ceil(total / limit);
-  const page = skip / limit + 1;
 
   return (
     <>
       <Row>
         <Col xs="12" md="9">
           <h1>DAG Blocks</h1>
         </Col>
-        <Col>
```
[the remark this answers is not in the document]
Comment (cblage) [reply pages/dag_blocks.js:60]: Discussed with Justin and Steve, and we decided to simplify the whole DAG page

File: pages/dag_blocks.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (cblage) [reply pages/dag_blocks.js:67]: actually, giving it a second look there's no more `reverse` var after simplifying the UI, so its always assumed to be `true`

[the remark this answers is not in the document]
Comment (rattrap) [reply pages/dag_blocks.js:67]: Ok. Then maybe we can remove the prop from the `DAG` component altogether ?

[the remark this answers is not in the document]
Comment (cblage) [reply pages/dag_blocks.js]: 07836f6

**Pull request merged**